pkgs/development/tools/database/sqlitebrowser/default.nix

```diff
@@ -1,5 +1,6 @@
-{ mkDerivation, lib, fetchFromGitHub, cmake, antlr
-, qtbase, qttools, qscintilla, sqlite }:
+{ mkDerivation, lib, fetchFromGitHub, qmake, antlr, qscintilla
+, qtbase, qttools, sqlite
```

Comment:
Did you mean to move `qscintilla` up?

Reply:
nope. in fact `sqlitebrowser` provides its own `qscintila` for the build process (but not even the old CMake derivation used this.
But you're absolutely right, I'll revert this to make the diff easier to read.